csm/conf/configure.py

```diff
@@ -110,9 +111,11 @@ async def execute(self, command):
                        f"{e} - {str(traceback.format_exc())}")
             Log.error(err_msg)
             raise CsmSetupError(err_msg)
+        Log.info("csm_setup: configure phase completed.")
         return Response(output=const.CSM_SETUP_PASS, rc=CSM_OPERATION_SUCESSFUL)
 
     def _prepare_and_validate_confstore_keys(self):
+        Log.info("Preparing and validating configuration store keys")
```

Comment:
Preparing and validating configuration store keys -> Config: Validating required configuration.

Reply:
updated